tests/src/topology/configs/time.rs

```diff
@@ -19,5 +27,9 @@ pub fn default_time_config() -> GeneralTimeConfig {
     GeneralTimeConfig {
         slot_duration: Duration::from_secs(slot_duration),
         chain_start_time: OffsetDateTime::now_utc(),
+        ntp_server: String::from("185.251.115.30"), // 0.europe.pool.ntp.org
```

Comment:
doesnt it work with ? I remember using some dns solver for this 🤔 
```suggestion
        ntp_server: String::from("0.europe.pool.ntp.org")
```

Reply:
This was disabled due to the random failures triggered by using the same listening port for all requests. This has now been solved and updated.